README.md

```diff
@@ -18,33 +36,29 @@ This new Custom role along with the built-in "Reader" role is applied to the sub
 3. Run the following
 ```bash
 curl -s https://raw.githubusercontent.com/cloudconformity/temp-azure-custom-policy/master/apply-roles | bash /dev/stdin \
-    --application-id <App registration client id> \
-    --subscription-id <Subscription id>
+    --application-id fj63c3b9-46e3-41ca-bf79-344a076fa9b2
```

Comment:
🤔 Shouldn't we specify a placeholder instead of an actual value (`fj63c3b9-46e3-41ca-bf79-344a076fa9b2`)?

Reply:
Yeah a good point, have updated